code/game/machinery/cryopod.dm

```diff
@@ -463,6 +463,10 @@
 			if(do_after(user, 20, target = G:affecting))
 				if(!M || !G || !G:affecting) return
 
+				if(src.occupant)
+					user << "<span class='notice'><B>\The [src] is in use.</B></span>"
```

Comment:
Might want to use `<span class='boldnotice'>` for both of these, rather than having both a span and `<B>`.